Compilation fails at src/ButtonWrapSizer.cpp: `'Node' is not a member of 'wxSizerItemList'`

* wxMaxima version 20.12.11
* GCC 10.2.1
* wxWidgets 3.1.4
* System: openSUSE Tumbleweed (Linux x86_64)
* Full cmake command used to configure: `
/usr/bin/cmake /home/abuild/rpmbuild/BUILD/wxmaxima-Version-20.12.1/. -GNinja -DCMAKE_INSTALL_PREFIX:PATH=/usr -DINCLUDE_INSTALL_DIR:PATH=/usr/include -DLIB_INSTALL_DIR:PATH=/usr/lib64 -DSYSCONF_INSTALL_DIR:PATH=/etc -DSHARE_INSTALL_PREFIX:PATH=/usr/share -DCMAKE_INSTALL_LIBDIR:PATH=lib64 -DCMAKE_INSTALL_LIBEXECDIR=/usr/libexec -DCMAKE_BUILD_TYPE=RelWithDebInfo '-DCMAKE_C_FLAGS=-O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -DNDEBUG' '-DCMAKE_CXX_FLAGS=-O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -DNDEBUG' '-DCMAKE_Fortran_FLAGS=-O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -DNDEBUG' '-DCMAKE_EXE_LINKER_FLAGS=-flto=auto -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now' '-DCMAKE_MODULE_LINKER_FLAGS=-flto=auto -Wl,--as-needed' '-DCMAKE_SHARED_LINKER_FLAGS=-flto=auto -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now' -DLIB_SUFFIX=64 -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=ON -DBUILD_STATIC_LIBS:BOOL=OFF -DCMAKE_COLOR_MAKEFILE:BOOL=OFF -DCMAKE_INSTALL_DO_STRIP:BOOL=OFF -DCMAKE_MODULES_INSTALL_DIR=/usr/lib64/cmake/wxMaxima`

```
[   28s] FAILED: src/CMakeFiles/wxmaxima.dir/ButtonWrapSizer.cpp.o 
[   28s] /usr/bin/ccache /var/lib/build/ccache/bin/c++ -DWXUSINGDLL -D_FILE_OFFSET_BITS=64 -D__WXGTK__ -I../src -I../src/cells -I../src/graphical_io -I../src/wizards -Isrc -I../data/winrc -isystem p
[   28s] ../src/ButtonWrapSizer.cpp: In member function 'virtual void Buttonwrapsizer::RecalcSizes()':
[   28s] ../src/ButtonWrapSizer.cpp:44:25: error: 'Node' is not a member of 'wxSizerItemList'
[   28s]    44 |   for (wxSizerItemList::Node *node = children.GetFirst(); node; node = node->GetNext())
[   28s]       |                         ^~~~
[   28s] ../src/ButtonWrapSizer.cpp:44:31: error: 'node' was not declared in this scope
[   28s]    44 |   for (wxSizerItemList::Node *node = children.GetFirst(); node; node = node->GetNext())
[   28s]       |                               ^~~~
[   28s] ../src/ButtonWrapSizer.cpp:60:25: error: 'Node' is not a member of 'wxSizerItemList'
[   28s]    60 |   for (wxSizerItemList::Node *node = children.GetFirst(); node; node = node->GetNext())
[   28s]       |                         ^~~~
[   28s] ../src/ButtonWrapSizer.cpp:60:31: error: 'node' was not declared in this scope
[   28s]    60 |   for (wxSizerItemList::Node *node = children.GetFirst(); node; node = node->GetNext())
[   28s]       |                               ^~~~

```

--- a/src/ButtonWrapSizer.cpp
+++ b/src/ButtonWrapSizer.cpp
@@ -41,7 +41,7 @@ void Buttonwrapsizer::RecalcSizes()
   wxSizerItemList children = GetChildren();
   long width = -1;
   long height = 20;
-  for (wxSizerItemList::Node *node = children.GetFirst(); node; node = node->GetNext())
+  for (auto node = children.GetFirst(); node; node = node->GetNext())
   {
     wxSizerItem* current =  node->GetData();
     wxWindow *item = current->GetWindow();
@@ -57,7 +57,7 @@ void Buttonwrapsizer::RecalcSizes()
 //    width = m_availSize / (m_availSize / width);
      
   wxSize bestSize(width, height);
-  for (wxSizerItemList::Node *node = children.GetFirst(); node; node = node->GetNext())
+  for (auto node = children.GetFirst(); node; node = node->GetNext())
   {
     wxSizerItem* current =  node->GetData();
     wxWindow *item = current->GetWindow();

